meson.build

```diff
@@ -0,0 +1,182 @@
+project(
+  'cloud-init',
+  meson_version: '>=0.63.0', # rockylinux/9
+  license: 'GPL-3 or Apache-2.0',
+  default_options: [
+    # The default can yield broken results.
+    'python.install_env=auto',
+  ],
+)
+system = host_machine.system()
+sysconfdir = get_option('sysconfdir')
+INIT_SYSTEM = get_option('init_system')
+
+lib_exec_dir = join_paths(get_option('prefix'), get_option('libexecdir'), 'cloud-init')
+
+pymod = import('python')
+python = pymod.find_installation('python3')
+
+#https://mesonbuild.com/FAQ.html#but-i-really-want-to-use-wildcards for downsteam drop-in files
+find = find_program('find')
+
+install_subdir(
+  'cloudinit',
+  install_dir: python.get_install_dir(),
+  install_tag: 'python',
+)
+
+completions = dependency('bash-completion')
```

Comment:
This variable is only used once, we could fold this dependency import into the function that uses it.

```meson
bash_completions_dir = dependency('bash-completion').get_variable(
...
```